doc/contributing/reviewing-contributions.xml

```diff
@@ -49,7 +49,7 @@
   <itemizedlist>
    <listitem>
     <para>
-     Add labels to the pull request. (Requires commit rights)
+     Add labels to the pull request. (Requires maintainer rights)
```

Comment:
s/maintainer/triage/

The maintainer privilege set is different from the triage privilege set on GitHub (see: https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level).

Reply:
That might be the case, but I think the language of github permissions is too low-level for this doc. Users shouldn’t need to understand the low level github permissions system, and how that maps on to teams in the NixOS org. The important question is, “what governs whether a user can add labels to PRs in the NixOS org?”, in language the user understands and can take action on.

I think it might be as simple as “all package maintainers” or “all NixOS org members” (“but who gets to be an org member?” is the natural question arising here)

Reply:
Why not both? That way, we can keep a high level user facing explanation ("NixOS org terminology"), and make it clear to people who know about GitHub rights _exactly_ what's going on. :smiley:

Reply:
The nixpkgs manual is already way too long and unfocused. I don’t want to add to that. 

We should maybe document the GitHub permissions thing somewhere, but this isn’t the doc section to do it.